website/integrations/services/jellyfin/index.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Jellyfin
+---
+
+<span class="badge badge--secondary">Support level: Community</span>
+
+## What is Jellyfin
+
+From https://jellyfin.org
+
+:::note
+Jellyfin is a free and open source media management and streaming platform for movies, TV shows, and music. 
+:::
+
+:::note
+Jellyfin does not have any native external authentication support as of the writing of this page.
+:::
+
+:::note
+Currently there are two plugins for Jelyfin that provide external authenticaion, an OIDC plugin and an LDAP plugin. This guide focuses on the use of the LDAP plugin.
+:::
+
+:::warning
+An LDAP outpost must be deployed to use the Jellyfin LDAP plugin
+:::
+## Preparation
+
+The following placeholders will be used:
+
+-   `jellyfin.company.com` is the FQDN of the Jellyfin install.
+-   `authentik.company.com` is the FQDN of the authentik install.
+-   `ldap.company.com` the FQDN of the LDAP outpost.
+-   `dc=company,dc=com` the Base DN of the LDAP outpost.
+-   `ldap_bind_user` the username of the desired LDAP Bind User
+
+## Service Configuration
+
+1. If you don't have one already create an LDAP bind user before starting these steps. 
+   - Ideally, this user doesn't have any permissions other than the ability to view other users. However, some functions do require an account with permissions.
+   - This user must be part of the group that is specified in the "Search group" in the LDAP outpost.
+2. Navigate to your Jellyfin installation and log in with the admin account or currently configured local admin.
+3. Open the administrator dashboard and go to the "Plugins" section. 
+4. Click "Catalog" at the top of the page, and locate the "LDAP Authentication Plugin"
+5. Install the plugin. You may need to restart Jellyfin to finish installation.
+6. Once finished navigate back to the plugins section of the admin dashboard, click the 3 dots on the "LDAP-Auth Plugin" card, and click settings.
+7. Configure the LDAP Settings as follows:
+   - ``LDAP Server``: ``ldap.company.com``
+   - ``LDAP Port``: 636
+   - ``Secure LDAP``: **Checked**
+   - ``StartTLS``: Unchecked
+   - ``Skip SSL/TLS Verification``: 
+     - If using a certificate issued by a certificate authority Jellyfin trusts, leave this unchecked. 
+     - If you're using a self signed certifcate, check this box.
+   - ``Allow password change``: Unchecked
+     - Since Authentik already has a frontend for password resets, its not nessessary to include this in Jellyfin, especially since it requires bind user to have privileges.
```

Comment:
Nit: https://goauthentik.io/developer-docs/docs/writing-documentation#general-guidelines, should be lowercase

Reply:
Fixed